build.rs

```diff
@@ -2,9 +2,60 @@ extern crate bindgen;
 
 use std::env;
 use std::path::PathBuf;
+use std::process::Command;
+
+fn build_static_espeak_ng() {
+    let out_dir = env::var("OUT_DIR").unwrap();
+    let espeak_dir = format!("{}/espeak-ng", out_dir);
+    let espeak_build_dir = format!("{}/build", &espeak_dir);
+    let espeak_lib_dir = format!("{}/lib", &espeak_build_dir);
+
+    Command::new("git")
+        .args([
+            "clone",
+            "https://github.com/espeak-ng/espeak-ng",

```

Comment:
We should probably use tags rather than the latest master to avoid breaking down stream users if something changes up-stream (could be a feature?)